Integrations/integration-Sandblast.yml

```diff
@@ -242,7 +242,27 @@ script:
                     maliciousData[uniqueKeys[key].toUpperCase()] = raw.Contents.response[uniqueKeys[key]];
                 }
             }
+            setContext('DBotScore', {'Indicator': raw.Contents.response.md5, 'Type': 'hash', 'Vendor': 'SandBlast', 'Score': 3, 'SubType': 'av'});
```

Comment:
why do you use setContext? I don't think we should use it any more, but rather return the context values in the entry